### PR TITLE
Remove motion components from fybrik repository

### PR DESCRIPTION
**Pull request by froesef**
Once the motion code is moved to a separate repository (https://github.com/fybrik/data-movement-operator) it can be removed from the main fybrik repository.

### CODE REVIEW

**Review by roee88 — APPROVED** [review]

File: manager/main.go [hunk 1]
```diff
@@ -47,15 +45,14 @@ var (
 )
 
 func init() {
-	_ = motionv1.AddToScheme(scheme)
 	_ = appv1.AddToScheme(scheme)
 	_ = corev1.AddToScheme(scheme)
 	_ = kbatch.AddToScheme(scheme)
 	_ = kapps.AddToScheme(scheme)
```
Comment (shlomitk1) [inline manager/main.go:51]: So we no longer need kbatch and kapps, do we?

**Comment by shlomitk1** [comment]
@froesef Is there any documentation on using copy operators with fybrik, in data-movement-operator repository?

**Review by shlomitk1 — APPROVED** [review]
Please add documentation in the data-movement-operator repo how to move data with fybrik (installation of the data-movement-operator helm chart, installing module yamls).

File: manager/main.go [hunk 2]
```diff
@@ -76,13 +73,7 @@ func run(namespace string, metricsAddr string, enableLeaderElection bool,
...
 		&corev1.ConfigMap{}:             {Field: systemNamespaceSelector},
 		&appv1.Blueprint{}:              {Field: systemNamespaceSelector},
-		&motionv1.BatchTransfer{}:       {Field: workerNamespaceSelector},
-		&motionv1.StreamTransfer{}:      {Field: workerNamespaceSelector},
-		&kbatch.Job{}:                   {Field: workerNamespaceSelector},
-		&kbatch.CronJob{}:               {Field: workerNamespaceSelector},
 		&corev1.Secret{}:                {Field: workerNamespaceSelector},
-		&corev1.Pod{}:                   {Field: workerNamespaceSelector},
-		&kapps.Deployment{}:             {Field: workerNamespaceSelector},
 		&corev1.PersistentVolumeClaim{}: {Field: workerNamespaceSelector},
```
Comment (shlomitk1) [inline manager/main.go:77]: I believe PVC was there only because of batch/streamtransfer.

**Pull request merged**